### PR TITLE
Change gen_fake_data command to also make CoursePrice objects

### PR DESCRIPTION
**Pull request by singingwolfboy**
Because `CoursePrice` objects are necessary to make the application work. :(

### CODE REVIEW

**Comment by noisecapella** [comment]
We should just get rid of `gen_fake_data` IMO. `gen_realistic_search_data` also exists and covers its use cases. Can you remove that management command and also update `gen_realistic_search_data` to add `CoursePrice` as you did here for `gen_fake_data`?

**Comment by gsidebo** [comment]
on top of that, can you change this to only add a CoursePrice if one doesn't already exist? you could use get_or_create

File: courses/management/commands/gen_fake_data.py [hunk 1]
```diff
@@ -53,6 +62,9 @@ def handle(self, *args, **options):
                 course_runs_range = range(randint(1, int(options['course_runs'])))
                 for _ in course_runs_range:
                     record_count += 1
-                    CourseRunFactory.create(course=course)
+                    run = CourseRunFactory.create(course=course)
+                    if options['course_prices']:
+                        CoursePriceFactory.create(course_run=run)
```
Comment (noisecapella) [inline courses/management/commands/gen_fake_data.py]: We should also set `is_valid=True` here. Also, all course runs need to have the same price

Comment (noisecapella) [inline courses/management/commands/gen_fake_data.py]: All course runs for a single program I mean

**Comment by singingwolfboy** [comment]
Closing in favor of #1463

**Pull request closed without merging**